recipes/event-model/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.2" %}
+
+package:
+    name: event-model
+    version: {{ version }}
+
+source:
+    url: https://github.com/NSLS-II/event-model/archive/v{{ version }}.tar.gz
+    fn: event-model-v{{version}}.tar.gz
+    sha256: b64f3297f7eaa3c67871074a745365596c1e32ae0d5b39ed3b55e956656cf4f0
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+
+    run:
+        - python
+        - enum34  # [py2k]
+
+test:
+    imports:
+        - event_model
+#    requires:
+#        - pytest
+#    commands:
+#        - py.test {{ environ.SRC_DIR }}
```

Comment:
Do you plan on using this at some point? If not let's just remove the commented line.